assets/src/edit-story/components/library/mediaLibrary.js

```diff
@@ -30,41 +46,143 @@ const Title = styled.h3`
 	font-size: 19px;
 	line-height: 20px;
 	line-height: 1.4em;
-	flex: 2 0 0;
+	flex: 3 0 0;
 `;
 
 const Header = styled.div`
 	display: flex;
-	margin: 0px 4px 10px;
+	margin: 0px 0px 25px;
 `;
 
 const Message = styled.div`
-	 color: ${ ( { theme } ) => theme.colors.fg.v1 };
+	color: ${ ( { theme } ) => theme.colors.fg.v1 };
 	font-size: 19px;
 `;
 
+const FilterButtons = styled.div`
+	border-bottom:  2px solid ${ ( { theme } ) => theme.colors.mg.v1 };
+	padding: 18px 0px;
+	margin: 10px 0px 15px;
+`;
+
+const FilterButton = styled.button`
+	border: 0px;
+	background: none;
+	padding: 0px;
+	margin: 0px 28px 0px 0px;
+	color: ${ ( { theme } ) => theme.colors.mg.v1 };
+	font-size: 13px;
+	${ ( { active } ) => active && ` color: #fff; font-weight: bold; ` };
+`;
+
+const Search = styled.input`
+	width: 100%;
+	background: ${ ( { theme } ) => theme.colors.mg.v1 } !important;
+	border-color: ${ ( { theme } ) => theme.colors.mg.v1 } !important;
+	color: ${ ( { theme } ) => theme.colors.mg.v2 } !important;
+	padding: 2px 10px 2px 33px !important;
+	&::placeholder {
+	  color: ${ ( { theme } ) => theme.colors.mg.v2 };
+	}
+`;
+
+const SUPPORTED_IMAGE_TYPES = [
+	'image/png',
+	'image/jpeg',
+	'image/jpg',
+	'image/gif',
+];
+
+const SUPPORTED_VIDEO_TYPES = [
+	'video/mp4',
+];
+
+const FILTERS = [
+	{ filter: '', name: 'All' },
+	{ filter: 'image', name: 'Images' },
+	{ filter: 'video', name: 'Video' },
```

Comment:
Looks like there's a separate filtering tag for `gif` in Figma and looks like `gif` file type is allowed already above. That filter will be implemented later, I suppose?

Reply:
I found that a little confusing, as gifs are images. It is almost a sub filter. The design for media library is changing. But I want to land this in, even at this stage as it unblocks other pieces of work like video element.